src/app/shared/directives/base-drag-and-drop/base-drag-and-drop.directive.ts

```diff
@@ -0,0 +1,37 @@
+import { Directive, EventEmitter, HostBinding, HostListener, Output } from '@angular/core';
+
+@Directive({
+  selector: '[appBaseDragAndDrop]',
+  standalone: true,
+})
+export class BaseDragAndDropDirective {
```

Comment:
Since it's about files, maybe we should have `file` in the name too?
```suggestion
export class FileDragAndDropDirective {
```

Reply:
I agree